sandbox/dev_tools/views.py

```diff
@@ -30,7 +30,7 @@ def dev_consumer(request: HttpRequest) -> HttpResponse:
                     "forum.lti.view",
                     kwargs={
                         "uuid": uuid.uuid5(
-                            uuid.NAMESPACE_DNS, form.cleaned_data["course_id"]
+                            uuid.NAMESPACE_DNS, form.cleaned_data["launch_url"]
```

Comment:
```python
"uuid": form.cleaned_data["forum_lti_id"]
```

Reply:
The `uuid.uuid5()` call is no longer necessary here. See my suggestion :arrow_up: